Library/Homebrew/dependency_collector.rb

```diff
@@ -136,7 +136,7 @@ def parse_class_spec(spec, tags)
   end
 
   def ant_dep(spec, tags)
-    if MacOS.version >= :mavericks
+    if MacOS.version >= :mavericks || !OS.mac?
```

Comment:
🆒! To be clear, I am excited about more stuff like this moving upstream. I still dream of a day when Linuxbrew/brew isn't needed.